### ISSUE TITLE
RegisterSymbolAction with SymbolKind.Parameter is incorrectly reported as unsupported

### ISSUE THREAD

**Issue by bkoelman** [dotnet/roslyn#16214]
**Version Used**: 
Microsoft.CodeAnalysis 2.0.0-rc2

**Steps to Reproduce**:
In VS 2017 RC, create an analyzer project with the following code:
```csharp
        public override void Initialize(AnalysisContext context)
        {
            context.RegisterSymbolAction(AnalyzeParameter, SymbolKind.Parameter);
        }

        private void AnalyzeParameter(SymbolAnalysisContext context)
        {
            throw new System.NotImplementedException();
        }
```

**Expected Behavior**:
No warnings, because the limitation no longer applies.

**Actual Behavior**:
```
Warning	RS1003	SymbolKind 'Parameter' is not supported for symbol analyzer actions.
```

With https://github.com/dotnet/roslyn/pull/13931 merged, this warning should be removed as suggested [here](https://github.com/dotnet/roslyn/pull/13931#issuecomment-251510515).

The latest prerelease version of Microsoft.CodeAnalysis.Analyzers (causing the warning) is still at 1.2.0-beta2.
